### ISSUE TITLE
Axios cache just stopped working with today's update - How to solve?

### ISSUE THREAD

**Issue by altcointrading** [kuitos/axios-extensions#95]
I am using Axios extensions linked from unpkg CDN in a vanilla JS snippet. The snippet was working for nearly a month. With no changes in template or in the script itself, suddently Axios extensions stopped working so I think it is related to the updates.

### Snippet

```
function getTickerAsync() {
const e = axios.create({
baseURL: "https://data.example.net",
adapter: window["axios-extensions"].cacheAdapterEnhancer(axios.defaults.adapter, {
    enabledByDefault: !0
})
});
e.get("/data.json").then(function(e) {
  document.getElementById.........
})
}
```

### Error 

1. First error is saying that my Axios call only has a get method, but the .del problem refers to a line on the inside of the Axios extension, there is no .del method used in my snippet. So, is there a workaround to make this work again? 
2. Second says that cacheAdapterEnhancer is not a function. Again, this worked for nearly a month.

<img width="662" alt="Screenshot 2022-10-05 at 6 02 09 PM" src="https://user-images.githubusercontent.com/17308419/194165523-3a34e54d-00c2-4b54-8908-b17f553b1939.png">

**Comment by govarthananve** [comment]
Please downgrade version 3.1.5 to 3.1.4 It will work

**Comment by kuitos** [comment]
fixed and released v3.1.6
